dao/src/main/java/se/fortnox/reactivewizard/util/JsonUtil.java

```diff
@@ -0,0 +1,55 @@
+package se.fortnox.reactivewizard.util;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectReader;
+import com.fasterxml.jackson.databind.type.TypeFactory;
+import com.fasterxml.jackson.databind.util.StdDateFormat;
+
+import java.lang.reflect.Type;
+import java.util.Map;
+
+public class JsonUtil {
```

Comment:
This should be dropped and our JsonDeserializerFactory should be used.

Reply:
I've managed to get rid of all but the convertValue method, is there any existing implementation that replaces that one?

Reply:
Just @Inject an ObjectMapper and use that.